src/index.js

```diff
@@ -354,7 +349,7 @@ class FocusHandlerImpl extends React.Component {
   }
 
   requestFocus = node => {
-    if (!this.state.shouldFocus) {
+    if (!this.state.shouldFocus && node) {
```

Comment:
in the tests the node was often not defined. I am not sure why, so I hope this doesn't cause further issues..